backend/fixtures/elections.sql

```diff
@@ -827,7 +827,54 @@ VALUES (1, 'Municipal Election', 'Heemdamseburg', 100, 'Municipal', '2024-11-30'
            ]
          }
        ]'),
-       (4, 'Filled Election', 'Heemdamseburg', 100, 'Municipal', '2024-11-30', '2024-11-1',
+       (4, 'Filled Election', 'Heemdamseburg', 100, 'Municipal', '2024-11-30', '2024-11-1', 'DataEntryInProgress',
+        '[
+          {
+            "number": 1,
+            "name": "Political Group A",
+            "candidates": [
+              {
+                "number": 1,
+                "initials": "A.",
+                "first_name": "Alice",
+                "last_name": "Foo",
+                "locality": "Amsterdam",
+                "gender": "Female"
+              },
+              {
+                "number": 2,
+                "initials": "C.",
+                "first_name": "Charlie",
+                "last_name": "Doe",
+                "locality": "Rotterdam",
+                "gender": null
+              }
+            ]
+          },
+          {
+            "number": 2,
+            "name": "Political Group B",
+            "candidates": [
+              {
+                "number": 1,
+                "initials": "T.",
+                "first_name": "Test",
+                "last_name": "Test",
+                "locality": "Amsterdam",
+                "gender": "Female"
+              },
+              {
+                "number": 2,
+                "initials": "B.",
+                "first_name": "Bar",
+                "last_name": "Foo",
+                "locality": "Rotterdam",
+                "gender": null
+              }
+            ]
+          }
+        ]'),
+       (5, 'Finished Election', 'Heemdamseburg', 100, 'Municipal', '2024-11-30', '2024-11-1', 'DataEntryFinished',
```

Comment:
This election (id 5) has no polling stations and no polling station results, even though it has as status `DataEntryFinished`. I don't think we shoud have incomplete test data like this.

I discovered this by going to by going to the status page of election 5, where it currently says "Steminvoer klaar (eerste zitting)" but also "Er zijn nog geen stembureaus toegevoegd voor deze verkiezing.".

Reply:
pair-fixed